tst/CTA.WebForms2Blazor.Tests/Extensions/SemanticModelExtensionTests.cs

```diff
@@ -117,7 +117,7 @@ public async Task GetNamespacesReferencedByType_Does_Not_Contain_Own_Namespace()
             var classSymbol = model.GetDeclaredSymbol(testClass1Declaration);
             var requiredNamespaces = model.GetNamespacesReferencedByType(testClass1Declaration);
 
-            Assert.False(requiredNamespaces.Contains(classSymbol?.ContainingNamespace));
+            Assert.False(requiredNamespaces.Contains(classSymbol?.ContainingNamespace.ToDisplayString()));
```

Comment:
Can this be null?

Reply:
I don't think we want to allow `classSymbol.ContainingNamespace` to be null here, as we want to assume that any class symbol we are processing belongs to a namespace, so we can probably get rid of the null conditional operator here, though I don't think `classSymbol.ContainingNamespace` should evaluate to null in the first place

Reply:
According to docs, ContainingNamespace can be null. It's possible to define classes without a namespace block. I believe in those cases, ContainingNamespace would be null.

Reply:
Ah ok, makes sense, thanks for clearing that up